includes/admin/discounts/discount-actions.php

```diff
@@ -142,6 +141,11 @@ function edd_admin_add_discount( $data = array() ) {
 		? 'discount_added'
 		: 'discount_add_failed';
 
+	if ( ! empty( $created ) ) {
+		// Now re-prime the transient for the has_acitve_discounts check
```

Comment:
```suggestion
		// Now re-prime the transient for the has_active_discounts check.
```
Typo in each instance of this comment--

Reply:
Fixed in https://github.com/awesomemotive/easy-digital-downloads/pull/9549/commits/61e661223acf9be5e7f9919354166b21621a5973